packages/app-runtime/test/lib/natives/FakeNativeNotificationAccess.ts

```diff
@@ -1,38 +1,38 @@
 import { ILogger } from "@js-soft/logging-abstractions";
 import { Result } from "@js-soft/ts-utils";
-import { INativeNotificationAccess, INativeNotificationScheduleOptions } from "../../src";
+import { INativeNotificationAccess, INativeNotificationScheduleOptions } from "../../../src";
 
-export class NativeNotificationAccessMock implements INativeNotificationAccess {
+export class FakeNativeNotificationAccess implements INativeNotificationAccess {
     public constructor(private readonly logger: ILogger) {}
 
     public schedule(title: string, body: string, options?: INativeNotificationScheduleOptions): Promise<Result<number>> {
-        this.logger.info(`NativeNotificationAccessMock.schedule(${title},${body},${options})`);
+        this.logger.info(`NativeNotificationAccess.schedule(${title},${body},${options})`);
 
         return Promise.resolve(Result.ok(0));
     }
 
     public update(id: number, title: string, body: string, options?: INativeNotificationScheduleOptions): Promise<Result<void>> {
-        this.logger.info(`NativeNotificationAccessMock.update(${id},${title},${body},${options})`);
+        this.logger.info(`NativeNotificationAccess.update(${id},${title},${body},${options})`);
```

Comment:
The "Fake" needs to be added before "NativeNotificationAccess", right? Also for the other log entries.

Reply:
Naah

Reply:
Why not? :D

Reply:
It's just some random log that noone will ever see.

Reply:
No one will see it, but we will know it exists 😝